### ISSUE TITLE
Remaining `image_util`

### ISSUE THREAD

**Issue by ysBach** [aplpy/aplpy#364]
The commit 699edb9c9d98996b15f58fde5690030dc9fcb63c tried to replace the previous `image_util`.
There is one line which still contains ``image_util``:
```python 
# core.py, line 924-928
        if type(levels) == int:
            auto_levels = image_util.percentile_function(image_contour)
            vmin = auto_levels(0.25)
            vmax = auto_levels(99.75)
            levels = np.linspace(vmin, vmax, levels)
```

**Comment by astrofrog** [comment]
Good catch, thanks! Feel free to fix this if you have time, otherwise I'll try and get to it soon.

**Comment by ysBach** [comment]
@astrofrog I am not a programmer and hope it is a right way to do it :)

I did not add any test since the Beginner Tutorial of APLPy can serve as the tutorial.
I checked the image (contour) from the updated version looks similar to the website's figure:
![myfirstplot](https://user-images.githubusercontent.com/17214293/31254279-e9eedd76-aa62-11e7-9c1e-1fddc28c01e7.png)
Trivial differences may have caused by the rendering issue.

One that I am worried about is that the beginner tutorial takes a lot of time:
```python
import aplpy
import numpy
import time

start = time.time()

gc = aplpy.FITSFigure('fits/2MASS_k.fits')
gc.show_rgb('graphics/2MASS_arcsinh_color.png')

gc.tick_labels.set_font(size='small')

gc.show_contour('fits/mips_24micron.fits', colors='white')

data = numpy.loadtxt('data/yso_wcs_only.txt')
ra, dec = data[:, 0], data[:, 1]

gc.show_markers(ra, dec, layer='marker_set_1', edgecolor='red',
                facecolor='none', marker='o', s=10, alpha=0.5)

gc.save('myfirstplot.png')

elapsed = time.time() - start
print(elapsed)
# 46.32778000831604
```
It is much slower that I expected..! :O

**Comment by astrofrog** [comment]
Fixed in #365
